sdk/healthcareapis/Microsoft.Azure.Management.HealthcareApis/src/Generated/Models/ServicesDescription.cs

```diff
@@ -18,7 +18,7 @@ namespace Microsoft.Azure.Management.HealthcareApis.Models
     /// <summary>
     /// The description of the service.
     /// </summary>
-    public partial class ServicesDescription : Resource
+    public partial class ServicesDescription : ServicesResource
```

Comment:
Additional breaking change

Reply:
This changes is approved by the ARM team. Only model definition has been renamed here from Resource -> ServicesResource.